.github/workflows/workflow.yml

```diff
@@ -116,15 +120,17 @@ jobs:
   samples:
     name: Build Samples
     needs: sdk
-    runs-on: macos-11
+    runs-on: macos-12
     continue-on-error: ${{ matrix.experimental }}
     strategy:
       fail-fast: false
       matrix:
-        xcode: [ '12.4' ]
+        xcode: [ '13.1' ]
```

Comment:
Should we use the `${{ env.XCODE_VERSION }}` here?

Reply:
That's make sens, I will update PR